### ISSUE TITLE
Channel closed after first message with bot.ListenForWebhook

### ISSUE THREAD

**Issue by zvejnmlz** [go-telegram-bot-api/telegram-bot-api#498]
https://github.com/go-telegram-bot-api/telegram-bot-api/blob/3f07c5f67644dfc8a9c37210de2b95cfdb060515/bot.go#L487

```
updates := bot.ListenForWebhook("/" + bot.Token)
for update := range updates {
    fmt.Println(update.UpdateId)
}
```

**Comment by Syfaro** [comment]
Oh yikes, that's not correct. I definitely overlooked some issues in the PR that made those changes and will get a fix out later today.

**Comment by Syfaro** [comment]
Should be fixed in https://github.com/go-telegram-bot-api/telegram-bot-api/commit/fde58e56ae1e2629ee6b4a77a7feb0ddf38340ee.

**Comment by Syfaro** [comment]
Released in [v5.5](https://github.com/go-telegram-bot-api/telegram-bot-api/releases/tag/v5.5.0)
